Analysis/Tasks/PID/pidTOF.cxx

```diff
@@ -39,19 +39,19 @@ void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
 
 #include "Framework/runDataProcessing.h"
 
-struct tofPid {
+struct tofPidTiny {
```

Comment:
Why keep the tiny here?

Reply:
You're right, I missed it ;) I will remove it